### PR TITLE
location and map functions

### PR DESCRIPTION
**Pull request by BeauHibbert**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+  } catch (error) {
+    console.error(error);
+    console.log('there was an error');
+    this.setState({ error: true })
+  }
+}
+getMap = async() => {
+  try {
+    console.log(this.state.locationObject.lat)
+    let result = await axios.get(`https://maps.locationiq.com/v3/staticmap?key=pk.7ce20acb246bf3b79f66fcb7d5ae9d3e&center=${this.state.locationObject.lat},${this.state.locationObject.lon}&zoom=1`);
```
Comment (RivaD2) [inline src/App.js:30]: Good work on the lab so far!

When I go to the deployed site, I am seeing that the `mapObject` is a bunch of encoded characters. Ensure that all API keys are added to `.env` and used as `process.env.REACT_API_KEY` inside url (see first key in url `key=pk.7ce` etc.).

File: src/App.js [hunk 2]
```diff
@@ -21,6 +49,13 @@ handleSubmit = (e) => {
           <input type="text" placeholder="city name" name="city" />
           <button type="submit">Explore!</button>
         </form>
+        <div>
+        {this.state.locationObject.display_name? <p>{this.state.locationObject.display_name}</p> : <p>Search for a city to explore</p>}
+        {this.state.locationObject.lat? <p>Latitude: {this.state.locationObject.lat}</p> : ""}
+        {this.state.locationObject.lon? <p>Longitude: {this.state.locationObject.lon}</p> : ""}
+        {this.state.mapObject? <div>{this.state.mapObject.data}</div> : ""}
```
Comment (RivaD2) [inline src/App.js:56]: What is showing for `mapObject.data` in console.log?

File: src/App.js (hunk 1 above)
Comment (Kellen-Linse) [reply src/App.js:30]: To make your map display you don't actually need to make a get request, you can simply input the URL into the img tags src attribute. I would suggest setting the URL to a variable first, so you can do something like this: 

img src="imageURL"

Check the "Static Map" Section of the docs page on the locationIQ website for more details.

Comment (RivaD2) [reply src/App.js:30]: Is the url correct here? The key in url on line 30,  should come from the .env file👍 When you have time, see screenshot I attached in Canvas response (that is what I am seeing for mapObject.data)

**Pull request merged**